packages/cli/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ All notable changes to the Zowe CLI package will be documented in this file.
 
 ## `8.7.0`
 
+-Enhancement: Added new command zowe zos-files download all-members-matching, (zowe files dl amm), to download members matching specified pattern(s). The success message for the Download.allMembers API was changed from originally "Data set downloaded successfully" to "Member(s) downloaded successfully." The change also alters the commandResponse when using the --rfj flag. [#2359](https://github.com/zowe/zowe-cli/pull/2359)
+
+## Recent Changes
```

Comment:
For new changelog items, I believe we want to move them towards the tope of the file. 
So, above the `## 8.8.0`  you want to have the `## Recent Changes` followed by the enhancement/bugFix entry 😋 
(Same way the changelog in zosfiles was modified) 😋